secure_uninstall/wizard_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<openerp>
+    <data>
+
+
+<record id="view_uninstall_wizard_form" model="ir.ui.view">
+    <field name="model">uninstall.check.wizard</field>
+    <field name="arch" type="xml">
+        <form string="Check" version="7.0">
+            <p class="oe_inline oe_grey">
+Uninstall module process remove all data managed by the module.
+            </p>
```

Comment:
maybe we should explain that "some columns can be dropped and there is no way to recover that except restoring a database backup". 

I know users don't read messages, but still... I do read them :smiley_cat: